resources/test/tox.ini

```diff
@@ -14,4 +14,4 @@ deps =
   six
   requests
 
-commands = pytest -vv {posargs}
+commands = pytest -vv {posargs} tests/functional/api-tests-1.html
```

Comment:
This will make it only run this file.

Reply:
Reverted - was a local change to speed up my testing.